hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -3822,6 +3824,26 @@ private void getProcedureResult(long procId, CompletableFuture<Void> future, int
     return future;
   }
 
+  @Override
+  public CompletableFuture<Map<String, Long>> updateCompactionServerTotalThroughput(long upperBound,
+      long lowerBound, long offPeak) {
+    CompletableFuture<Map<String, Long>> future = this.<Map<String, Long>> newMasterCaller().action(
+      (controller, stub) -> this.<UpdateCompactionServerTotalThroughputRequest,
+        UpdateCompactionServerTotalThroughputResponse, Map<String, Long>> call(
+            controller, stub, UpdateCompactionServerTotalThroughputRequest.newBuilder()
+                .setMaxThroughputUpperBound(upperBound).setMaxThroughputLowerBound(lowerBound)
+                .setMaxThroughputOffPeak(offPeak).build(),
+          (s, c, req, done) -> s.updateCompactionServerTotalThroughput(c, req, done), resp -> {
+            Map<String, Long> result = new HashMap<>();
+            result.put("UpperBound", resp.getMaxThroughputUpperBound());
```

Comment:
Why return the result like this? The comment says "the now total throughput of all compaction servers", so I thought the key of the map is the name of a compaction server...

If we just want to return 3 long values, why not introduce a new class?